deployment/conf/.templates/deploy.cfg.templ

```diff
@@ -14,6 +14,8 @@ mongodb-pwd = {{ default .Env.mongodb_pwd "" }}
 
 # The KBase auth server url.
 auth-service-url = {{ default .Env.auth_service_url "https://kbase.us/services/authorization/Sessions/Login" }}
+# The KBase auth API.
+auth-service-api = {{ default .Env.auth_service_url "https://ci.kbase.us/services/auth" }}
```

Comment:
It's currently set to auth_service_url=https://ci.kbase.us/services/auth/api/legacy/KBase/Sessions/Login